interface/.dockerignore

```diff
@@ -0,0 +1,4 @@
+bin
+build
+node_modules
```

Comment:
Oh sorry @rzelayafavila I didn't see your comments when replying. To summarize this thread:
* We don't need the `node_modules` directory anymore. These should be `rm -r`'d from your machines and dropped from the `.dockerignore` file.
* This `Dockerfile` should be split into two, adding a `Dockerfile.base` which contains all dependencies and is hosted in Dockerhub. This `Dockerfile` should then be based off of that one and probably not do too much more than copy the code in and run `grunt`.

Does that seem accurate?